### ISSUE TITLE
/register command asks for email and password, not key

### ISSUE THREAD

**Issue by tmad40blue** [yariplus/nodebb-plugin-minecraft-integration#101]
Hello, when I attempt to use `/register [key]` in-game, I am presented with the message:

`Error: Please use '/register [email] [password]`

instead of being asked for a key. If I attempt to use my forum email and password, I am presented with the message:

`Data error. Please inform an administrator.`

Nothing is printed to either the Minecraft server log OR the NodeBB log when either of these occurs.

Your assistance with registering Minecraft users with forum users is greatly appreciated.

**Comment by yariplus** [comment]
Sure thing! What version of the Minecraft plugin are you using?

**Comment by tmad40blue** [comment]
@yariplus 

I actually just solved the issue. The version of the plugin available at https://dev.bukkit.org/projects/nodebb-integration - which is the very first result that hosts a download of the Minecraft server plugin - is very, very out of date. I only found your Github "releases" page by arbitrarily searching through other solved issues on Github.

With the latest release from Github everything works swimmingly.

Perhaps the actual issue is that the BukkitDev version of the plugin is out of date.
